misc/iitthesis.cls

```diff
@@ -333,19 +329,27 @@
  }{ }
 }
 
+% Biblatex general settings
+\RequirePackage[isbn=false, doi=true, url=true, maxbibnames=99, language=american,
```

Comment:
1. Why is maxbibnames=99?
2. Why is maxalphanames=4?
3. Why should we have DOI in our printed bibliographies?

Reply:
`maxalphanames` is needed to get the citations similar to what was the default before. It means it would take up to 4 initials to create the citation key. The rest are just my preferences that I copied over. In many cases publishers asked to keep all author names and DOIs, so that's what I use.

I don't know if DOI is that helpful on printed theses, but it can help locate references quickly when using a pdf.